WebApiCoreSeed.UnitTests/Infrastructure/AuthZero/AuthZeroServiceTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿namespace WebApiCoreSeed.UnitTests.AuthZero
+{
+    using Auth0.Core;
+    using Auth0.Core.Exceptions;
+    using Auth0.ManagementApi;
+    using Auth0.ManagementApi.Clients;
+    using Auth0.ManagementApi.Models;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Moq;
+    using Newtonsoft.Json.Linq;
+    using RestSharp;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading.Tasks;
+    using WebApiCoreSeed.Infrastructure.AuthZero;
+    using WebApiCoreSeed.Infrastructure.Result;
+    using System.Net.Http;
+
+    [TestClass]
+    public class AuthZeroServiceTests
+    {
+        const string clientId = "hky5bSNB8PJhOsD4P6lJn1JdBybI3aG0";
+        const string clientSecret = "3eBRn_P5kPb5d5HdAx4U9Kxkif5DOgRn8mAK9bKW3e-j7krurgwrD1cICOO9QCEM";
+        const string domain = "webapicoreseed.auth0.com";
+        const string newUserEmail = "noreply@makingsense.com";
```

Comment:
I know these are unit tests and the interaction with the services are mocked, but these values are all fake, right? We shouldn't be including any real-life credentials in the code.